packages/ws-server/src/webSocketServer.ts

```diff
@@ -169,6 +169,16 @@ app.ws.use(async (ctx) => {
             connectionIdPrefix,
           );
           break;
+        case WS_CONSTANTS.METHODS.ETH_GET_CODE:
+          relayResponse = await relay.eth().getCode(params[0], params[1]);
```

Comment:
Why didn't you use the `response` variable init at line 131? Using the same `response`you wouldn't have to send the result back to the client again yourself but down below (after the switch block) it already send back to the client. 

```suggestion
const contractCode = await relay.eth().getCode(params[0], params[1]);
response = jsonResp(request.id, null, contractCode)
```

Reply:
If I do that it falls into the if statement on line 194, which will not format the response correctly, or at least match the ethereum RPC's response:
```
    if (response) {
      ctx.websocket.send(JSON.stringify(response));
    }
```